.mergify.yml

```diff
@@ -0,0 +1,10 @@
+---
+pull_request_rules:
+  - name: Automatic merge on approval
+    conditions:
+      - "approved-reviews-by>=1"
+      - "check-success=Semantic Pull Request"
+      - "check-success=symplegma-crio:linter"
+    actions:
+      merge:
+        method: merge
```

Comment:
Could you change to squash. Can you do it in the already merge other repo ?

Reply:
sure!